docs/token-guides/cross-chain-tokens-guide.md

```diff
@@ -0,0 +1,462 @@
+# Lido cross-chain tokens adoption guide
+
+:::warning Disclaimer
+This guide provides recommendations provided by the [Network Expansion Committee (NEC)](https://snapshot.org/#/lido-snapshot.eth/proposal/0x7cdf1af7cfeb472ae202c45fb6d7e952bb34bfcbc82113549986b2bc2d5f54c5). Following these recommendations increases the likelihood of recognition by the Committee, but does not guarantee it. Moreover, the Lido DAO vote, with a quorum established, can override any NEC decision at any time, even if it has already been implemented and released. Therefore, NEC makes no warranties, express or implied, and disclaims all implied warranties, including any warranty of the likelihood of the recognition or rejection by the Lido DAO.
+:::
+
+## TL;DR
+
+If you are looking for a typical copy-paste solution, refer to the suitable subsection below if any fits your case.
+Follow the [General Scenario Towards Recognition](#general-scenario-towards-the-recognition).
+
+### Ethereum L2 built on OP-Stack, wstETH + stETH
+
+Use [multichain-automaton](https://github.com/lidofinance/multichain-automaton) for automated deployment of the wstETH + stETH setup. This tool will automatically deploy the default wstETH + stETH setup, which satisfies the required [Recommendations](#recommendations).
+
+Please follow recommendation R-9, which is specific to stETH bridging.
+
+### Ethereum L2, wstETH
+
+Deploy the setup from the [lido-l2](https://github.com/lidofinance/lido-l2) repository. This repository allows you to deploy the default wstETH setup, which satisfies the required [Recommendations](#recommendations).
```

Comment:
can we list stacks and branches for them here?

Reply:
- the branch is main is up-to-date, so I think no need to mention branch
- added that there are setups for OP-stack and Arbitrum